client/src/components/TestSuite/TestSuiteUtilities.tsx

```diff
@@ -33,3 +33,14 @@ function getInputsRecursive(testGroup: TestGroup, testOutputs: Set<string>): Tes
   testGroup.outputs.forEach((output) => testOutputs.add(output.name));
   return inputs;
 }
+
+export const shouldShowDescription = (
+  runnable: Test | TestGroup | TestSuite,
+  description: JSX.Element | undefined
+): boolean => {
+  if (description && runnable.description && runnable.description.length > 0) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
Just return the top conditional

Reply:
That was my original but the linter was complaining but I'll try again.

Reply:
I've tried every permutation I can think of but typescript thinks this is impossible.  `Type 'undefined' is not assignable to type 'boolean'` 🤷

Reply:
Very weird, I'll take a look